database/exip.sql

```diff
@@ -593,12 +593,13 @@ CREATE TABLE `Declaration` (
   `agreeToAntiBribery` tinyint(1) DEFAULT NULL,
   `agreeToConfidentiality` tinyint(1) DEFAULT NULL,
   `agreeToConfirmationAndAcknowledgements` tinyint(1) DEFAULT NULL,
-  `application` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `application` varchar(191) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci DEFAULT NULL,
```

Comment:
should this have changed?

Reply:
Yes, it now matches what a SQL dump export has. Seems to have been missed.